### ISSUE TITLE
💅 useValidAriaValues false positive for svelte aria-rowcount attribute using variable

### ISSUE THREAD

**Issue by minht11** [biomejs/biome#10637]
### Environment information

<details>

```bash

CLI:
  Version:                      2.5.0
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

```

</details>

### Rule name

useValidAriaValues

### Playground link

https://biomejs.dev/playground/?lineWidth=120&expand=always&whitespaceSensitivity=strict&indentScriptAndStyle=true&tab=formatter&pane=Diagnostics&files.main.svelte=PABzAGMAcgBpAHAAdAAgAGwAYQBuAGcAPQAiAHQAcwAiAD4ACgAgACAAYwBvAG4AcwB0ACAAYwBvAHUAbgB0ACAAPQAgADEACgA8AC8AcwBjAHIAaQBwAHQAPgAKAAoAPABkAGkAdgAKAAkACQByAG8AbABlAD0AIgBnAHIAaQBkACIACgAJAAkAYQByAGkAYQAtAHIAbwB3AGMAbwB1AG4AdAA9AHsAYwBvAHUAbgB0AH0ACgA%2BAAoAPAAvAGQAaQB2AD4ACgA%3D

### Expected result

Should not report "The value of the ARIA attribute aria-rowcount is not correct." for when value is a variable.
```svelte
<script lang="ts">
  const count = 1
</script>

<div

		role="grid"
		aria-rowcount={count}
></div>

```

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by yzakmuz** [comment]
Hey, I'd like to work on the issue can you assign me?

**Comment by Netail** [comment]
> Hey, I'd like to work on the issue can you assign me? 

Hi @yzakmuz,
There's [a PR](https://github.com/biomejs/biome/pull/10683) open already. It crosses over multiple rules and frameworks, which are all tackled in the PR. It's almost done. Only thing left in the PR is static values using a vue binding (`v-bind:foor="'bar'"`), which I am finishing up.
